template/guestbook_message_template.tpl.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/**
+ * @file
+ * Default theme implementation for guestbook messages.
+ *
+ * Available variables:
+ * - $variables['body']: an array containing the following information:
+ *   - 'id' - message id;
+ *   - 'uid' - user id;
+ *   - 'sid' - session id;
+ *   - 'date' - unix timestamp;
+ *   - 'message' - message text;
+ *   - 'user_name' - author of message;
+ *   - 'formatted_date' - formatted unix timestamp.
+ *
+ * @see theme_pager()
+ */
+?>
+<?php  global $user; ?>
```

Comment:
@PavelSPN pass variables through preprocess functions, do not set it directly inside template